asm/istio/options/envoy-access-log.yaml

```diff
@@ -16,7 +16,10 @@
 ---
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
+metadata:
+  name: envoy-access-log
 spec:
   meshConfig:
     accessLogFile: "/dev/stdout"
+    # [END docs]
```

Comment:
Is this needed?

Reply:
Same for above 4 files.

Reply:
It seems like this is used to generate docs. I don't know how, but I think we should assume they can't just be removed.